### PR TITLE
Fixes overlapping footer bug.

### LINKED ISSUE #436: UI bug: Half visible "Download as PDF button"
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

After generating the badges the download button appears. If I submit a request for another batch of badges and preview them, then the download button and footer overlaps.

**Expected behavior:**

The users should be redirected to another page where they can download the badges.

**Steps to reproduce:**
1. Generate the badges
2. Enter data for another request and then preview it.

image

### PR DESCRIPTION
**Pull request by rupav**
Fixes #436 

#### Checklist

- [ x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ x] My branch is up-to-date with the Upstream `development` branch.
- [ x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Fixes overlapping css footer problem with "download pdf" element and guide page.
- Added correct path for vagrant setup for the shell provisioning. (As I used vagrant to do the changes and it was required).

**Earlier**:

**After the fix**:

### CODE REVIEW

**Comment by rupav** [comment]
@vaibhavsingh97 @abishekvashok @gabru-md please review.

File: v1/app/static/css/style.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 .board .btn-success {
     margin-top: 10px;
-    margin-bottom: 10px;
+    margin-bottom: 50px;
```
Comment (abishekvashok) [inline v1/app/static/css/style.css]: Pls change the front-end and not the v1. It's only as a backup.

Comment (rupav) [reply v1/app/static/css/style.css]: @abishekvashok done similarly in the front-end folder.

Comment (abishekvashok) [reply v1/app/static/css/style.css]: Pls remove the changes done in v1,
Sorry if I wasn't crystal clear last time :) @rupav

File: Vagrantfile [hunk 2]
```diff
@@ -66,5 +66,5 @@ Vagrant.configure(2) do |config|
   # Enable provisioning with a shell script. Additional provisioners such as
   # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
   # documentation for more information about their specific syntax and use.
-  config.vm.provision :shell, :path => "scripts/install.sh"
+  config.vm.provision :shell, :path => "v1/scripts/install.sh"
```
Comment (abishekvashok) [inline Vagrantfile]: Also undo this change

**Comment by rupav** [comment]
@abishekvashok I think changes should be reflected in v1 too. Because later on if we take it as a reference for replicating other files in backend and front-end, we may directly copy the content, thus might be losing some info, for e.g. the css modifications above. Also if changes are correct, then why shouldn't we modify v1 too. Please correct me if I am wrong here.

**Comment by abishekvashok** [comment]
> Also if changes are correct, then why shouldn't we modify v1 too.

We won't be deploying v1. It's just a backup of the last stable release. We shouldn't be modifying the last stable release right? We can go either way, at the end the implementors decide which way to go :),

**Comment by rupav** [comment]
@abishekvashok done.

**Comment by abishekvashok** [comment]
@rupav thank you for working on it :)
